capabilities/display_capabilities.js

```diff
@@ -388,87 +388,7 @@ SDL.templateCapabilities = {
             },
             "numCustomPresetsAvailable": 10
         },
-        "buttonCapabilities": [{
-            "name": "PRESET_0",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_1",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_2",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_3",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_4",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_5",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_6",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_7",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_8",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PRESET_9",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "OK",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "PLAY_PAUSE",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "SEEKLEFT",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "SEEKRIGHT",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "TUNEUP",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }, {
-            "name": "TUNEDOWN",
-            "shortPressAvailable": true,
-            "longPressAvailable": true,
-            "upDownAvailable": true
-        }],
+        "buttonCapabilities": SDL.ButtonCapability.filter(button => button.name !== 'CUSTOM_BUTTON'),
```

Comment:
```suggestion
        "buttonCapabilities": SDL.ButtonCapability,
```
No need to omit `CUSTOM_BUTTON`

Reply:
@jacobkeeler As you can see, `CUSTOM_BUTTON` is ommited [in develop branch](https://github.com/smartdevicelink/sdl_hmi/blob/d58988ec3e1d00f619bcb88d68f5e3aff2fec312/capabilities/display_capabilities.js#L391)
Are you sure that is should be ommited?

Reply:
see this comment: https://github.com/smartdevicelink/sdl_hmi/pull/580#discussion_r684246053

Reply:
@jacobkeeler Implemented 5563ddf80f22964bb8696a1735d8f2604accec06